tests/modules/images/regenerate-existing-images/background-process/perflab-background-job-test.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * Tests for background-job module.
+ *
+ * @package performance-lab
+ * @group   regenerate-existing-images
+ */
+
+/**
+ * Class Perflab_Background_Job_Test
+ *
+ * @coversDefaultClass Perflab_Background_Job
+ * @group regenerate-existing-images
+ */
+class Perflab_Background_Job_Test extends WP_UnitTestCase {
+
+	/**
+	 * Job instance.
+	 *
+	 * @var Perflab_Background_Job
+	 */
+	private $job;
```

Comment:
Should be remove.

```suggestion
```

Reply:
@mukeshpanchal27 We are using this property across various methods.

Reply:
Ok. Thanks.